orca/templates/_common/resources/usr/local/bin/fetch_orca_secrets.sh

```diff
@@ -15,12 +15,20 @@ function check_usage {
 }
 
 function get_token {
-	local token=$(curl --fail --request POST --silent "http://${ORCA_VAULT_ADDRESSES}/v1/auth/userpass-${1}/login/${1}" --data '{"password": "'${ORCA_VAULT_SERVICE_PASSWORD}'"}')
+	while true;
```

Comment:
This might actually turn into an infinite look. Can you please also add the counter here?

The ; is not necessary after the true. :)

Reply:
> The ; is not necessary after the true. :)
That's an old pattern of me, it makes the code more readable, actually.
Removed.

Reply:
> This might actually turn into an infinite look. Can you please also add the counter here?
I did it this way, because there are other infinite loops was well.
I will add a counter.

Reply:
Counter added.